opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/SpanBuilderShim.java

```diff
@@ -197,7 +197,7 @@ public Span start() {
       builder.setNoParent();
       baggage = Baggage.empty();
     } else if (mainParent != null) {
-      builder.setParent(Context.root().with(io.opentelemetry.api.trace.Span.wrap(mainParent)));
+      builder.setParent(Context.root().with(PropagatedOpenTelemetrySpan.create(mainParent)));
```

Comment:
parents in OT are expressed via SpanContexts, not full spans, so why do we need PropagatedOpenTelemetrySpan?